test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/GroupMemberGrsControllerSpec.scala

```diff
@@ -113,14 +108,55 @@ class GroupMemberGrsControllerSpec extends ControllerSpec {
 
         membersListSize mustBe 1
       }
+
+      "registering UK Limited Company with same company number as Nominated organisation" in {
+        authorizedUser()
+        mockGetUkCompanyDetails(
+          IncorporationDetails("12345678",
+                               testCompanyName,
+                               testUtr,
+                               testBusinessVerificationPassStatus,
+                               testCompanyAddress,
+                               incorporationRegistrationDetails
+          )
+        )
+        mockRegistrationFind(registeredLimitedCompany)
+        mockRegistrationUpdate()
+
+        await(controller.grsCallback(registeredLimitedCompany.incorpJourneyId.get)(getRequest()))
+
+        val membersListSize: Boolean = getLastSavedRegistration.groupDetail.get.members.isEmpty
+
+        membersListSize mustBe true
```

Comment:
The variable name is a little misleading, perhaps just use `getLastSavedRegistration.groupDetail.get.members.isEmpty mustBe true`?

Reply:
Done